src/shell/notifications.test.js

```diff
@@ -0,0 +1,88 @@
+import { screen } from '@testing-library/react'
+import { within } from '@testing-library/dom'
+import userEvent from '@testing-library/user-event'
+
+import { endDay, getItemByName } from '../test-utils/ui'
+import { STORM_MESSAGE } from '../strings'
+import { farmhandStub } from '../test-utils/stubs/farmhandStub'
+import { saveFileStubFactory } from '../test-utils/stubs/saveFileStubFactory'
+
+describe('notifications', () => {
+  test('pending notifications are shown when a day starts', async () => {
+    jest.spyOn(Math, 'random').mockReturnValue(0)
+
+    await farmhandStub()
+    await endDay()
+
+    // STORM_MESSAGE message is a sample pending day notification that is known
+    // to show upon starting a new day when Math.random() is locked to 0.
```

Comment:
would be great to capture this in a variable/function. like `STORM_CHANCE_VALUE = 0` used for the `mockReturnValue` above, or a `isStormingToday` util that can be mocked